flaky/flaky_pytest_plugin.py

```diff
@@ -251,6 +274,17 @@ def call(self, func, plugin):
         is_call = self.when == 'call'
         try:
             self.result = func()
+            # pytest's unittest plugin for some reason doesn't actually raise
+            # errors. It just adds them to the unittest result. In order to
+            # determine whether or not the test needs to be rerun, this
+            # code looks for the _excinfo attribute set by the plugin.
+            excinfo = getattr(self._item, '_excinfo', None)
+            if (
+                    excinfo is not None and
+                    isinstance(excinfo, list) and
```

Comment:
Doesn't this line make the previous line unnecessary?

You can also have ``excinfo`` default to ``[]``.